src/endpoint.ts

```diff
@@ -272,13 +272,14 @@ export class Endpoint<
         input: initialInput,
       });
     } catch (e) {
-      if (e instanceof Error) {
-        lastResortHandler({
-          logger,
-          response,
-          error: new ResultHandlerError(e.message, error),
-        });
-      }
+      lastResortHandler({
+        logger,
+        response,
+        error: new ResultHandlerError(
+          (e instanceof Error ? e : new Error(`${e}`)).message,
```

Comment:
```suggestion
          e instanceof Error ? e.message : `${e}`,
```

Reply:
Yeah, that definitely can be simplified, but I also had an idea of extracting the `unknown` exception handling into a function like this:

```ts
function makeErrorFromAnything(subject: unknown) {
  return e instanceof Error ? e.message : `${e}`;
}
```

so that stringifying behaviour would be kept in a single dedicated place and could be adjusted or even customized in the future.